docs/articles/documentation/extending-testcafe/custom-reporter/reporter-project-commands.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: docs
+title: Reporter Project Commands
+permalink: /documentation/extending-testcafe/custom-reporter-plugin/reporter-project-commands/
+---
+# Reporter Project Commands
```

Comment:
@inikulin, do you mind if I'll locate the _Using Development Version with TestCafe_ section between the_ Preview Report _and _Publish Reporter to NPM_ sections, not in the _Impementing Reporter_ section?